test/models/user_test.rb

```diff
@@ -0,0 +1,27 @@
+require "test_helper"
+
+describe User do
+
+  describe "validations" do
+    it "can be valid" do
+      is_valid = users(:valid_user).valid?
```

Comment:
You don't have a fixture nicknamed `valid_user`.

```suggestion
      is_valid = users(:valid_user1).valid?
```